.ci/.e2e-tests-for-elastic-agent.yaml

```diff
@@ -4,40 +4,53 @@ PLATFORMS:
     image: "ami-0d90bed76900e679a"
```

Comment:
Can you apply the very same variables to the `.e2e-tests.yaml` file? That one is the main descriptor, and this "elastic-agent" one is intended to be used by Beats PRs.